src/android/io/sentry/SentryCordova.java

```diff
@@ -14,7 +14,8 @@
 
 import java.util.Date;
 
-import io.sentry.android.AndroidSentryClientFactory;
+import io.sentry.android.core.SentryAndroid;
+import io.sentry.Sentry;
```

Comment:
this is not necessary right

Reply:
Oh is it not? I was following in what we had on RN